oak_containers_system_image/Dockerfile

```diff
@@ -3,6 +3,15 @@ FROM clearlinux:base
 # Remove .dockerenv as we will not run this image in an actual Docker container.
 RUN rm /.dockerenv
 
+# Install Clear Linux's containers-basic bundle. This gives an installation of
+# docker to run containers. This is a starting point only, we should switch to
+# a more lightweight runtime later.
+# Ref: https://clearlinux.github.io/clear-linux-documentation/tutorials/docker.html#install-the-containers-basic-bundle
+RUN swupd bundle-add containers-basic
```

Comment:
We cannot specify a release version for the bundle, since bundles are versions are mapped 1-1 to the distro release.